site/content/en/docs/Guides/counters-and-lists.md

```diff
@@ -0,0 +1,376 @@
+will succeed, assuming a GameServer is found.
+
+This can be combined with the ability to define multiple `selectors` to create quite sophisticated `GameServer` 
+selection options that can attempt to find the most appropriate `GameServer`.
+
+See the [Allocation Reference]({{< ref "/docs/Reference/gameserverallocation.md" >}}) for more details.
+
+The `priorities` block also gives you options to prioritise specific `GameServers` over others when performing an 
+Allocation, depending on the Allocation Strategy.
+
+* Packed: The [usual infrastructure optimisation strategy]({{< ref "/docs/Advanced/scheduling-and-autoscaling.md#allocation-scheduling-strategy" >}})
+  still applies, but the `priorities` block is used as a tie-breaker within the least utilised infrastructure, to ensure
+  optimal infrastructure usage while also allowing some custom prioritisation of `GameServers`.
+* Distributed: The entire selection of `GameServers` will be sorted by this priority list to provide the
+  order that `GameServers` will be allocated by.
+
+For example, if we wanted to select a `GameServer` that had:
+
+1. At least one `room` available in the GameServer capacity.
+2. First check Allocated `GameServers` and falling back to `Ready` `GameServers` if there aren't any.
+3. Include general infrastructure optimisation for least usage.
+4. And also ensure the `GameServers` that were most full are allocated first (after infrastructure optimisations
+   where applied).
+
+We could implement the following `GameServerAllocation`:
+
+```yaml
+apiVersion: allocation.agones.dev/v1
+kind: GameServerAllocation
+spec:
+  scheduling: Packed
+  # Choose most full `GameServers` first
+  priorities:
+    - type: Counter
+      key: rooms
+      order: Ascending
+  selectors:
+    # First check to see if we can back-fill an already allocated `GameServer`
+    - gameServerState: Allocated
+      matchLabels:
+        agones.dev/fleet: simple-game-server
+      counters:
+        rooms:
+          minAvailable: 1
+    # If we can't, then go get a `Ready` `GameServer`.
+    - gameServerState: Ready
+      matchLabels:
+        agones.dev/fleet: simple-game-server
+      counters:
+        rooms:
+          minAvailable: 1
+  counters:
+    rooms:
+      action: Increment
+      amount: 1
+```
+
+For more details on how Agones implements infrastructure optimisation, see the documentation on
+[Scheduling and Autoscaling]({{< ref "/docs/Advanced/scheduling-and-autoscaling.md" >}}).
+
+## Fleet Scale Down Prioritisation
+
+Another optimisation control you can apply with `Fleets` when Agones' scaled them down, is to also set `priorities` 
+to influence the order in which `GameServers` are shutdown and deleted.
+
+While neither `players` or `rooms` are particularly good examples for this functionality, if we wanted to ensure 
+that `Ready` `GameServers` with the most available capacity `rooms` where a factor when scaling down a `Fleet` we could 
+implement the following:
```

Comment:
@igooch this would be incorrect then? (Given some comment above?)

Reply:
Ah missed this. Yep, it refers to total capacity, not available.

Reply:
I added a warning here about the documentation being incorrect for now, but will be accurate soon.

PTAL- but I think that's the last outstanding item for this PR? Would love to have this live for GDC 😃